administrator/language/en-GB/plg_system_shortcut.ini

```
@@ -0,0 +1,12 @@
+; Joomla! Project
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SYSTEM_SHORTCUT="System - Keyboard Shortcut"
+PLG_SYSTEM_SHORTCUT_OVERVIEW_DESC="Press <span class=\"badge bg-dark px-2\">J</span> to access the shortkey mode followed by the shortkey."
+PLG_SYSTEM_SHORTCUT_OVERVIEW_HINT="Press J + X for our keyboard shortcut overview"
+PLG_SYSTEM_SHORTCUT_OVERVIEW_TITLE="Available Joomla Shortcuts"
```

```suggestion
PLG_SYSTEM_SHORTCUT_OVERVIEW_TITLE="Joomla Keyboard Shortcuts"
```

We an't say available as the shortcut might not be available on the page where this is displayed